source/io/module_propertyNode.js

```diff
@@ -0,0 +1,150 @@
+// Using a modified UMD module format. Specifically a modified returnExports (no dependencies) version
+(function (root, globalName, factory) {
+    'use strict';
+    var buildGlobalNamespace = function () {
+        var buildArgs = Array.prototype.slice.call(arguments);
+        return globalName.split('.').reduce(function (currObj, subNamespace, currentIndex, globalNameParts) {
+            var nextValue = currentIndex === globalNameParts.length - 1 ? factory.apply(undefined, buildArgs) : {};
+            return currObj[subNamespace] === undefined ? (currObj[subNamespace] = nextValue) : currObj[subNamespace];
+        }, root);
+    };
+
+    if (typeof define === 'function' && define.amd) {
+        // AMD. Register as a named module.
+        define(globalName, [], factory);
+    } else if (typeof module === 'object' && module.exports) {
+        // Node. "CommonJS-like" for environments like Node but not strict CommonJS
+        module.exports = factory();
+    } else {
+        // Browser globals (root is window)
+        buildGlobalNamespace();
+    }
+}(this, 'NationalInstruments.Vireo.ModuleBuilders.assignPropertyNode', function () {
+    'use strict';
+
+    /* global Map */
+
+    // Static Private Variables (all vireo instances)
+    var ERRORS = {
+        // Shared
+        NO_ERROR: {
+            CODE: 0,
+            MESSAGE: ''
+        },
+
+        kNIObjectReferenceIsInvalid: {
+            CODE: 1055,
+            MESSAGE: 'Object reference is invalid.'
+        }
+    };
+
+    var formatMessageWithException = function (messageText, exception) {
+        if (typeof exception.message === 'string' && exception.message.length !== 0) {
+            return messageText + ', <APPEND>\n ' + exception.message;
+        }
+
+        return messageText;
+    };
+
+    // Vireo Core Mixin Function
+    var assignPropertyNode = function (Module, publicAPI) {
+        // Disable new-cap for the cwrap functions so the names can be the same in C and JS
+        /* eslint 'new-cap': ['error', {'capIsNewExceptions': [
+            'NationalInstruments',
+            'DispatchMessagetToHTMLPanel'
+        ]}], */
+
+        Module.propertyNode = {};
+        publicAPI.propertyNode = {};
+
+        var readProperty = function () {
+            // Dummy no-op function.
+        };
+
+        var writeProperty = function () {
+            // Dummy no-op function.
+        };
+
+        // Private Instance Variables (per vireo instance)
+        Module.propertyNode.jsPropertyNodeWrite = function (
+            viNamePointer,
+            dataItemIdPointer,
+            propertyNamePointer,
+            propertyTypePointer,
+            propertyPathPointer,
+            errorStatusPointer,
+            errorCodePointer,
+            errorSourcePointer) {
+            /* eslint-disable no-undef */
```

Comment:
Why is this eslint disable here? Remove it or add an elaborate justification in a comment above it